### PR TITLE
Feature/release notes 10.6.0

### PR DESCRIPTION
**Pull request by ThomasRitense**

### CODE REVIEW

File: extending-valtimo/multi-instance-ready.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Making Valtimo multi instance ready
+
+It can be desirable to run multiple instances of the same Valtimo implementation. In that case, some additional steps
+are required to mak your Valtimo implementation multi instance ready.
```
Comment (theo-ritense) [inline extending-valtimo/multi-instance-ready.md:4]: @Klaas-Ritense *make

File: extending-valtimo/multi-instance-ready.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+In some cases, Valtimo saves a file to the local storage. All instances of Valtimo must be configured to use the same
+storage.
+
+## Shared directory for temporary files
+
+Besides having shared storage, all Valtimo instances must also be configured to use the same directory for temporary
+files. This can be set with the property: `valtimo.resource.temp.directory`
+
+Or environment variable: `VALTIMO_RESOURCE_TEMP_DIRECTORY`
```
Comment (theo-ritense) [inline extending-valtimo/multi-instance-ready.md:16]: @Klaas-Ritense I think this is still part of the previous sentence.

File: extending-valtimo/extending-valtimo.md [hunk 3]
```diff
@@ -16,3 +16,4 @@ code.
 * [Tabs](tabs/tabs.md)
 * [Integrating spring beans in a process](integrate-spring-bean-in-process.md)
 * [Creating a custom dashboard](creating-a-custom-dashboard.md)
+* [Make Valtimo multi instance ready](multi-instance-ready.md)
```
Comment (marijnritense) [inline extending-valtimo/extending-valtimo.md:19]:
```suggestion
* [Making Valtimo multi instance ready](multi-instance-ready.md)
```

File: SUMMARY.md [hunk 4]
```diff
@@ -99,9 +98,15 @@
     * [Portaal taak Plugin](using-valtimo/plugin/portaaltaak/configure-portaaltaak-plugin.md)
     * [Exact Plugin](using-valtimo/plugin/exact/configure-exact-plugin.md)
     * [Verzoek Plugin](using-valtimo/plugin/verzoek/configure-verzoek-plugin.md)
+    * [Besluiten Plugin](using-valtimo/plugin/besluiten-api/configure-besluiten-api-plugin.md)
   * [Process](using-valtimo/process/process.md)
     * [System processes](using-valtimo/process/systemprocesses.md)
     * [Correlating messages](using-valtimo/process/correlating-messages.md)
+  * [Process links](using-valtimo/process-link/process-link.md)
+    * [Create process links](using-valtimo/process-link/create-process-link.md)
```
Comment (marijnritense) [inline SUMMARY.md:106]:
```suggestion
    * [Creating process links](using-valtimo/process-link/create-process-link.md)
```

File: SUMMARY.md [hunk 5]
```diff
@@ -99,9 +98,15 @@
...
+    * [Edit process links](using-valtimo/process-link/edit-process-link.md)
```
Comment (marijnritense) [inline SUMMARY.md:107]:
```suggestion
    * [Editing process links](using-valtimo/process-link/edit-process-link.md)
```

File: SUMMARY.md [hunk 6]
```diff
@@ -99,9 +98,15 @@
...
+    * [Unlink process links](using-valtimo/process-link/unlink-process-link.md)
```
Comment (marijnritense) [inline SUMMARY.md:108]:
```suggestion
    * [Unlinking process links](using-valtimo/process-link/unlink-process-link.md)
```

**Review by marijnritense — APPROVED** [review]

**Pull request merged**